openspec/changes/enforce-proxy-timing-seams/notes.md

```
@@ -0,0 +1,59 @@
+- `python scripts/check_proxy_architecture.py`, `python scripts/check_cancellation_safety.py`: passed.
+- `pytest tests/unit/test_check_proxy_timing_seams.py tests/unit/test_check_proxy_architecture.py tests/unit/test_check_cancellation_safety.py`: 98 passed.
+- `ruff check scripts tests/unit/test_check_proxy_timing_seams.py`, `ruff format --check`: clean.
+- `ty check scripts/check_proxy_timing_seams.py tests/unit/test_check_proxy_timing_seams.py`: passed.
+- `openspec validate enforce-proxy-timing-seams --strict`: passed.
+
+## Negative proof
+
+Inserting `await asyncio.sleep(0.5)` after the `scheduler.sleep(chunk_seconds)`
+call in `app/modules/proxy/_service/http_bridge/streaming.py` produced:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)